Cargo.toml

```
@@ -47,3 +45,11 @@ rand_core = { version = "0.6", default-features = false }
 [target.'cfg(not(target_feature = "rdrand"))'.dependencies.rand]
 version = "0.8"
 features = ["std", "std_rng"]
+
+# Support running tests in wasm-pack
+[target.'cfg(all(any(target_arch = "wasm32", target_arch = "wasm64"), target_os = "unknown"))'.dev-dependencies]
+# When doing cargo test --target wasm32-unknown-unknown, or wasm-pack test, the build fails if getrandom/js is not enabled.
+# See https://docs.rs/getrandom/latest/getrandom/#webassembly-support for guidance
+getrandom = { version = "*", features = ["js"] }
```

```suggestion
getrandom = { version = "0.2", features = ["js"] }
```

Per [the docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html):

> Note: [crates.io](https://crates.io/) does not allow bare * versions.